stable/datadog/values.yaml

```diff
@@ -236,6 +241,22 @@ datadog:
     #
     port: 8126
 
+    ## @param useSocketVolume - boolean - optional
+    ## Enable APM over Unix Domain Socket
+    ## ref: https://docs.datadoghq.com/developers/dogstatsd/unix_socket/
+    #
+    useSocketVolume:  # true
+
+    ## @param socketPath - string - optional
+    ## Path to the trace-agent socket
+    #
+    socketPath: /var/run/datadog/apm.socket
```

Comment:
I think we don’t need to be able to customise the path of the socket *inside* the container.
Being able to customise the path of the socket on the socket is useful, but inside the container, we can hardcode à path for the mount point and the environment variable.

Reply:
yes but it was done like this for dogstatsd, that is why I wanted to have the same logic for apm